docs/helm-secrets.adoc

```diff
@@ -69,7 +69,6 @@ Note that if you used the link:user-installation.adoc[User Installation] instruc
 
 If you want to give additional people access to view and edit secrets, you can do so via the following:
 ```
-sops --decrypt secrets.yaml > secrets.dec.yaml
-sops --encrypt --age <your_age_public_key>,<another_age_public_key> secrets.dec.yaml > secrets.yaml
-rm secrets.dec.yaml
+sops -r -i --add-age <your_age_public_key> secrets.yaml
```

Comment:
This could say `another_age_public_key` instead of `your_age_public_key` to avoid confusion?